src/fairseq2/models/wav2vec2/builder.py

```diff
@@ -139,6 +139,9 @@ class Wav2Vec2EncoderConfig:
     depthwise_conv_kernel_size: int
     """The kernel size of depthwise convolutions in Conformer blocks."""
 
+    conv_norm_type: Literal["batch_norm", "layer_norm"]
```

Comment:
Unlike S2T Transformer and w2v-BERT, I assume you need "conv_norm_type" here for the new wav2vec2 architecture in SC. If that is the case, we can keep this, but otherwise, you can remove it similar to other models.

Reply:
Yes we do need it for the new wav2vec2 model.